ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/sflow.j2

```diff
@@ -41,6 +41,25 @@ sFlow is enabled.
 
 sFlow is disabled.
 {%     endif %}
+{%     if sflow.hardware_acceleration.enabled is arista.avd.defined(true) %}
+
+sFlow hardware acceleration is enabled.
+{%     endif %}
+{%     if sflow.hardware_acceleration.sample is arista.avd.defined %}
+
+sFlow hardware accelerated Sample Rate: {{ sflow.hardware_acceleration.sample }}
+{%     endif %}
+{%     if sflow.hardware_acceleration.modules is arista.avd.defined and sflow.hardware_acceleration.modules | length > 0 %}
+
+### SFlow Hardware Accelerated Modules
+| Module | Acceleration Enabled |
```

Comment:
Blank line required between heading (line54) and the table first row (line55)

Reply:
Or this will lead to MarkDown error later.

Reply:
done!